modular_chomp/code/modules/mob/living/simple_mob/subtypes/xenomorph.dm

```diff
@@ -95,33 +134,41 @@
 	src.adjust_nutrition(src.max_nutrition)
 	sight |= SEE_MOBS
 
+/mob/living/simple_mob/xeno_ch/Initialize(mapload)
+    .=..()
+    LoadComponent(/datum/component/squeak, list('modular_chomp/sound/effects/footstep/hardclaw1.ogg'), 50)
```

Comment:
You could use set_footsteps instead for them to use all claw footsteps when on different materials as well
```suggestion
    set_footsteps(FOOTSTEP_MOB_CLAW)
```